src/main/java/org/killbill/billing/plugin/vertex/VertexTaxCalculator.java

```diff
@@ -361,7 +373,12 @@ private SaleRequestType toTaxRequest(final Account account,
 
         SellerType sellerType = new SellerType();
         sellerType.setCompany(companyName);
-        sellerType.setDivision(companyDivision);
+
```

Comment:
we need to set taxregistration -> isoCountryCode as well

Reply:
added tax registration properties